gradle-plugin/src/main/resources/kotlin-ktor-client/api.mustache

```diff
@@ -52,7 +52,17 @@ internal class {{classname}}(basePath: kotlin.String = "{{{basePath}}}", httpCli
         {{#bodyParams}}
         if({{paramName}} != null) {
             builder.body = LargeTextContent(
-                _json.stringify({{dataType}}.serializer(),{{paramName}}),
+        {{#dataType}}
+            {{#isListContainer}}
+                {{#items}}
+                _json.stringify({{dataType}}.serializer().list,
+                {{/items}}
+            {{/isListContainer}}
+            {{^isListContainer}}
+                _json.stringify({{dataType}}.serializer(),
+            {{/isListContainer}}
+                {{paramName}}),
+        {{/dataType}}
```

Comment:
as i see this can be simplified:
```
_json.stringify({{dataType}}.serializer(){{#isListContainer}}.list{{/isListContainer}},
```
but i not understand now why here used {{#dataType}} and {{#items}} - it's necessary?

Reply:
all ok after discussion